app/services/CheckoutService.scala

```diff
@@ -22,10 +22,14 @@ class CheckoutService(identityService: IdentityService, salesforceService: Sales
       identityService.registerGuest(subscriptionData.personalData)
     }
 
+    //TODO when implementing test-users this requires updating to supply data to correct location
+    val touchpointBackend = TouchpointBackend.Normal
+    val ratePlan = touchpointBackend.ratePlan.ratePlanId
+
     for {
       idUser <- idUserF
       memberId <- salesforceService.createOrUpdateUser(subscriptionData.personalData, idUser)
```

Comment:
Yes - I didn't want to do it in this PR as it looks like a bit of refactoring is required.